docs/v13/views/create-new-prototype.html

```diff
@@ -0,0 +1,35 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+Create a new prototype
+{% endblock %}
+
+{% block beforeContent %}
+	{{ govukBreadcrumbs({
+	  items: [
+	    {
+	      text: "GOV.UK Prototype Kit",
+	      href: "./"
+	    }
+	  ]
+	}) }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl">Create a new prototype</h1>
+
+    <p>Choose from:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>create a prototype <a href="./install/getting-started">for new users</a></li>
```

Comment:
I think this is meant to link to `./install/create-a-prototype`?

At the minute if you follow the 'for new users' link from https://prototype-kit-docs-v13.herokuapp.com/docs/create-new-prototype the documentation assumes you already have a prototype created.

Reply:
Also wondering if the link text is sufficient to provide context for e.g. a screen reader user – should the whole list item be linked instead?

Reply:
I'm pretty sure this is linking to the right page. @NoraGDS can you confirm?

Reply:
that is the correct link

Reply:
OK. I still think users are missing some crucial information but if you're sure 👍🏻